brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Post Layout - Added show taxonomy above title and normal/highlighted taxonomy styling option in grid, masonry and carousel.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/120ab759-bb2c-40b2-a5f4-942908090d72?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- Testing demo in grid - https://share.getcloudapp.com/12ugDx51

### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Drag and drop grid, masonry, and carousel.
- Go to the content > Enable show taxonomy you will see 2 options Show taxonomy above title and when enabling this you will get again 2 options hide taxonomy icon and taxonomy style normal/highlighted.
- If you set taxonomy style highlighted then the styling option for highlighted layout comes in meta like this - https://share.getcloudapp.com/xQuq0KwX
- Demo Link - http://uag-development.ra7yia5yg3-wg96glqzy3oy.p.runcloud.link/post-layout-taxonomy-styling-option/

Test cases -
- New User
   - Drag & Drop Post block ( Carousel, Masonry & grid )
   - Go to the Content tab > Enable Show Taxonomy option. You will see Show taxonomy above title option Taxonomy styling ( Normal & Highlight )
   - Add multiple diff categories to post and check how it will be displayed on the front & editor as well.
   - Check the Show Meta Icon option.
- Backword -
   - Create a grid, masonry, and carousel with multiple categories and Show taxonomy enables.
   - Switch to branch & check editor and front.

### Checklist:
